incubator/hnc/go.mod

```diff
@@ -14,11 +14,11 @@ require (
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
-	github.com/pkg/errors v0.8.1
```

Comment:
Question: this probably was used in our /third-party code, so it's gone after using cert-controller directly?

Reply:
That's my assumption too. I generally just like `go mod tidy` or `go mod vendor` add and remove things without caring too much, unless I know I care about it deeply (e.g. replacing the version of `controller-runtime`).